examples/fairings/src/main.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(proc_macro_hygiene)]
+#![feature(const_type_id)]
```

Comment:
I can't seem to find a tracking issue for this feature (the unstable points to the catch-all https://github.com/rust-lang/rust/issues/57563).